site/_data/i18n/tags.yml

```diff
@@ -76,3 +82,9 @@ new-in-devtools:
 
 devtools-engineering:
   en: DevTools Engineering
+
+origin-trials:
```

Comment:
Understand the `chrome` tag is added on purpose. Are these tags as well?

Reply:
Yes, they're actually missing and we're rendering bad data where people have tagged articles with them D: